Moodlet/app/src/main/res/layout/fragment_add_mood.xml

```diff
@@ -184,7 +184,8 @@
                 android:layout_marginStart="16dp"
                 android:layout_marginLeft="16dp"
                 android:layout_marginTop="8dp"
-                android:text="Somebody touch spaghet"
+                android:maxLength="20"
+                android:hint="Reason"
```

Comment:
Please change this to "Somebody touch spaghet" and make it a string resource

Reply:
ok dad